lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -2195,372 +2160,558 @@ can be found below.
 The implementation of these tags are simply Python functions defined in the
 [/lib/galaxy/tool_util/verify/asserts](https://github.com/galaxyproject/galaxy/tree/dev/lib/galaxy/tool_util/verify/asserts)
 module.
-]]>
-      </xs:documentation>
+]]></xs:documentation>
     </xs:annotation>
     <xs:choice minOccurs="0" maxOccurs="unbounded">
-      <xs:group ref="TestAssertionsGeneral" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:group ref="TestAssertionsText" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:group ref="TestAssertionsTabular" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:group ref="TestAssertionsArchive" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:group ref="TestAssertionsXml" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:group ref="TestAssertionsJson" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:group ref="TestAssertionsH5" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:group ref="TestAssertionsImage" minOccurs="0" maxOccurs="unbounded"/>
+      <xs:group ref="TestAssertion"/>
```

Comment:
Is there a way to ensure that this part is not edited?

Reply:
I'll see if I can regenerated these with some comments.

Reply:
Also we could put the definitions in their own file. That would probably be a better design in the abstract.

Reply:
I've regenerated the docs with a lot more warnings about things being auto-generated and not to modify them. I've also rearranged things so the XSD has comments about where modules are defined so the docs can be updated.